repr.go

```diff
@@ -193,6 +195,11 @@ func (p *Printer) reprValue(seen map[reflect.Value]bool, v reflect.Value, indent
 	}
 	t := v.Type()
 
+	if t == rawJSONType {
+		fmt.Fprintf(p.w, "%s", v.Bytes())
```

Comment:
I like the idea, but I don't think this is what we want. This will make it impossible to copy+paste the output of repr and have it usable, which is one of the goals of this package.

Reply:
We probably want something like:

```go
fmt.Fprintf(p.w, "json.RawMessage(%q)", v.Bytes())
```

Reply:
I opted for `json.RawMessage(`%s`)` instead, as it gives easier output.

If you have a string with `json.RawMessage(%q)` you get `json.RawMessage(\"\\\"string\\\"\")` since the quotes need to be escaped. With the backtick wrapped `%s` formatter you avoid that.

Reply:
So, in my example, you now get:

```
Value: json.RawMessage(`"plain literal"`),
```

Whereas with `%q` you'd get:

```
Value: json.RawMessage("\"plain literal\""),
```